ui/library/src/components/PrintButton.tsx

```diff
@@ -0,0 +1,30 @@
+import * as React from 'react';
+
+export type Props = {
+  className?: string;
+  children?: React.ReactNode;
+};
+
+/**
+ * HTML anchor tag allows you to download a file from the same origin.
+ * This is a workaround to download a file served from a different origin
+ */
+export const PrintButton: React.FunctionComponent<Props> = ({
+  children,
+  className,
+  ...extraProps
+}: Props) => {
+  const onClickPrint = () => {
```

Comment:
Can you also export this function? Having the print functionality outside of the button will be great incase we want to make a different button

Reply:
no issue @ericmarsh995

Reply:
@ericmarsh995 updated

Reply:
thanks!